release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -115,6 +115,16 @@ For more information, see xref:../security/audit-log-policy-config.adoc#configur
 
 You can now customize the URL for the internal OAuth server. For more information, see xref:../authentication/configuring-internal-oauth.adoc#customizing-the-oauth-server-url_configuring-internal-oauth[Customizing the internal OAuth server URL].
 
+[id="ocp-4-9-security-tls-profile-setting"]
+==== Additional TLS security profile setting on the API server
+
+In OpenShift Container Platform 4.9, etcd is available as a control plane component.
```

Comment:
```suggestion
In {product-title} 4.9, etcd is available as a control plane component.
```

We don't hardcode in OpenShift Container Platform. So, use {product-title} attribute instead and it will automatically display the product title.